components/builder-originsrv/src/migrations/origin_packages.rs

```diff
@@ -267,5 +317,32 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
+    migrator.migrate("originsrv",
+                     r#"CREATE OR REPLACE FUNCTION get_origin_packages_for_version_v1 (
+                            op_ident text,
```

Comment:
I'd rename this to something like `op_ident_prefix` to reflect the fact that it's (potentially) a partial identifier.